src/components/Setup/Views/InstallPWA.tsx

```
@@ -13,170 +14,147 @@ import { useCallback, useEffect, useState } from 'react'
+    const [isLoadingLocal, setIsLoadingLocal] = useState(true)
+    const [showUnsupportedBrowserModal, setShowUnsupportedBrowserModal] = useState(false)
+
+    useEffect(() => {
+        if (typeof window !== 'undefined') {
+            setIsPWAInstalled(window.matchMedia('(display-mode: standalone)').matches)
+        }
+    }, [])
 
-    // Redirect to home if user is logged in
     useEffect(() => {
         if (!!user) push('/home')
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)